src/seed/seeder.ts

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
+/* eslint-disable */
```

Comment:
The seeding logic is using v5 of ethers. Since currently is not a priority to update this because there is the actual indexing logic, I would leave that as it is